tangelo/linq/tests/test_simulator.py

```diff
@@ -94,20 +107,20 @@ class TestSimulateAllBackends(unittest.TestCase):
 
     def test_get_exp_value_operator_too_long(self):
         """ Ensure an error is returned if the qubit operator acts on more qubits than are present in the circuit """
-        for b in installed_simulator:
+        for b in installed_simulator and installed_clifford_simulators:
```

Comment:
I think you want the union of these two sets. Here `and` most likely does something weird.

`for b in (installed_simulator | instaled_clifford_simulators):` is likely what you want I think